source/_layouts/base.liquid

```diff
@@ -45,7 +45,9 @@
     </script>
   </head>
 
-  <body data-layout="body">
+  {% assign classes = page.filePathStem | replace: "/", " " %}
```

Comment:
Looks great! The extra space isn't a big deal, but I think we could fix it pretty easily with:

```suggestion
  {%- assign classes = page.filePathStem | replace: '/', ' ' | strip -%}
```